packages/client/components/EditingStatus/EditingStatus.tsx

```diff
@@ -26,7 +26,10 @@ const StatusHeader = styled('div')({
 })
 
 const EditingText = styled('span')<{isEditing: boolean}>(({isEditing}) => ({
-  cursor: isEditing ? 'default' : 'pointer'
+  cursor: isEditing ? 'default' : 'pointer',
+  overflowWrap: 'break-word',
+  wordWrap: 'break-word',
+  inlineSize: 50
```

Comment:
Why do we need exactly `inlineSize` property? Would `max-width` or `width` work?